packages/itwin/grouping-mapping-widget/src/widget/components/QueryBuilder.ts

```diff
@@ -389,10 +389,10 @@ export class QueryBuilder {
     const baseClass = this.query.classes[0];
     const baseClassName = baseClass.className;
     const baseIdName = baseClass.isAspect
-      ? `${baseClassName}.Element.id ECInstanceId`
+      ? `${baseClassName}.Element.id`
       : `${baseClassName}.ECInstanceId`;
 
-    let queryString = `SELECT ${baseIdName} FROM ${baseClassName}`;
+    let queryString = `SELECT ${baseIdName}${baseClass.isAspect ? " ECInstanceId" : ""} FROM ${baseClassName}`;
```

Comment:
how is this any different?

Reply:
[test it out](https://www.typescriptlang.org/play?#code/MYewdgzgLgBAjgVwKYCcCeB5ANgExgXhgAoAjAQwiQGEsKIAuGAIQqQEoCA+GAbwFgAUDBihIscpRp0AcmQC2SAjAnVaECADpgaiLIUBuQcJHhoy1gEkcexYRVT1GgJYQAghAAOSYFCPHhAPwwAAYAJDz2OjYAvhoAolhICmBQznhxVBZiZGDASFbBfv6MYRGsDrrySLEZWdA5eQWGAkUwibCIqGgAylAoTmAA5krB3XEAMhkAKjDhKlYxMABiAEoYALKzZZJRVdHBzf6iECCJGlggg0Sd6L39Q2yC0YKCxx3I6NJIAO5KpOU6RgsSgcfDcfhCExicw7GRVJSROhaXYGIpvGH5azwuwApEudxeHytQIhOa49QxeKJZKpJw4QqQ4qk7aqOEKGqZbK5TEHF6MtpId5dO4DYaEUYTaZbeZY9lk2GOfGebywIIAIhgtS5jRwGsYarV0WWa028tZFL2vP5x1OSHOl2uHx6fVFjwEzxaAigaC8zFYSghwiVhKgjBIIFtOUOIhRSEY0Hug2aHteplgZADfmDKsYfWQABo-No2XGYGrXGqnnz0SRM5Dsz5cygC0XY-qmJX3XybphcEQyG6e19vv23YIe9gcKRB07h9OgA)